politeiawww/api/v1/v1.go

```diff
@@ -354,12 +367,13 @@ type Login struct {
 
 // LoginReply is used to reply to the Login command.
 type LoginReply struct {
-	IsAdmin        bool    `json:"isadmin"`        // Set if user is an admin
-	UserID         string  `json:"userid"`         // User id
-	Email          string  `json:"email"`          // User email
-	PublicKey      string  `json:"publickey"`      // Active public key
-	PaywallAddress string  `json:"paywalladdress"` // Registration paywall address
-	PaywallAmount  float64 `json:"paywallamount"`  // Registration paywall amount in DCR
+	IsAdmin            bool    `json:"isadmin"`            // Set if user is an admin
+	UserID             string  `json:"userid"`             // User id
+	Email              string  `json:"email"`              // User email
+	PublicKey          string  `json:"publickey"`          // Active public key
+	PaywallAddress     string  `json:"paywalladdress"`     // Registration paywall address
+	PaywallAmount      float64 `json:"paywallamount"`      // Registration paywall amount in DCR
```

Comment:
please no floats

Reply:
@marcopeereboom what should we be using instead?